picamera2/picamera2.py

```diff
@@ -873,8 +873,6 @@ def configure_(self, camera_config="preview") -> None:
         """
         if self.started:
             raise RuntimeError("Camera must be stopped before configuring")
-        if self.encoder is not None and self.encoder.running:
```

Comment:
Removed this check, to allow starting multiple encoders

Reply:
It feels to me as though we still want something like this. It would need to complain if _any_ encoder in the set is still running. Does that sound correct?